src/routes.js

```diff
@@ -1,17 +1,15 @@
 import React from "react";
-import { Router, Route, Link, IndexRoute, hashHistory, browserHistory } from 'react-router';
```

Comment:
I think we still need browserHistory to ensure navigation through the pages while preserving the history. 😄

Reply:
the browserHistory prop is already passed in to the router in index.js, it was duplicated so I removed the unnecessary instance in routes.js

Reply:
Cool